packages/app/src/widgets/header/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
Let's use different names for different files. `constants` for constants and `header` for headers. I mean, not only `index`.